src/SB/Game/zTalkBox.cpp

```diff
@@ -13,6 +13,22 @@ namespace
     SharedTalkboxState ztalkbox_shared;
 }
 
+void ztalkbox::reset_all() 
+{
+    volatile int* sharedMemory = reinterpret_cast<volatile int*>(ztalkbox_shared.active);
+    
+    int value = 0;
+    *sharedMemory = value;
+    *(sharedMemory + 8) = value;
+    *(sharedMemory + 9) = 0;
+    *(sharedMemory + 10) = 0;
+    *(sharedMemory + 11) = 0;
+    *(sharedMemory + 12) = 0;
+
+    clear_layout(); //FIXME: Find and replace with xtextbox::layout::clear()
+    return;
+}
```

Comment:
This is something I wouldn't want to include, it's very rough and only raises the function to 28%, so it's a big work in progress still, and doesn't necessarily give us anything useful to work off of. I'd remove it for now from this PR.